Extend CHANGELOG.md with a draft for a 2.0.0 release note (#1734)
OpenEnergyPlatform/ontology: CHANGELOG.md

## [2.0.0] - 2023-10-24

### Major stuctural changes
For the version 2.0.0 we especially did some major stuctural changes on the OEO. Two new modules were introduced:
* The first one, oeo-sector, is a module on the same hierarchical level as the other content-related modules, i.e. oeo-physical, oeo-model and oeo-social. All sector-related entities were moved there from oeo-social.
* The second one, oeo-shared-axioms, is a module that imports the four content-related modules and contains all axioms that are introduced across modules.

This caused a lot of movement of entities, especially from oeo-shared. Many classes were moved back to their original module. This restructuring was done to foster the modularity, which makes the implementation easier and more error-prone. Yet, we had to break with the rule, that all axioms should live in the same module as their class of declaration. The full axiomatisation of a class is now only provided if the module oeo-shared-axioms is used. See also the wiki article on the [modules of the OEO](https://github.com/OpenEnergyPlatform/ontology/wiki/Modules-of-the-OEO).

> Further, the import process for UO and OMO where updated. All scripts and tools not essential for the building routine moved to a [seperate repository](https://github.com/OpenEnergyPlatform/oeo-tools). And finally, english language labels were added to (english) definitions and labels, to allow other language extentions in the future.

Reviewer: ```suggestion
Further, the import process for UO and OMO were updated. All scripts and tools not essential for the building routine moved to a [seperate repository](https://github.com/OpenEnergyPlatform/oeo-tools). And finally, english language labels were added to (english) definitions and labels, to allow other language extentions in the future.
```

> ## [1.X.X] - 20XX-XX-XX

Reviewer: ```suggestion
## [2.0.0] - 2023-10-24
```

### Major stuctural changes
For the version 2.0.0 we especially did some major stuctural changes on the OEO. Two new modules were introduced:
* The first one, oeo-sector, is a module on the same hierarchical level as the other content-related modules, i.e. oeo-physical, oeo-model and oeo-social. All sector-related entities were moved there from oeo-social.
* The second one, oeo-shared-axioms, is a module that imports the four content-related modules and contains all axioms that are introduced across modules.

This caused a lot of movement of entities, especially from oeo-shared. Many classes were moved back to their original module. This restructuring was done to foster the modularity, which makes the implementation easier and more error-prone. Yet, we had to break with the rule, that all axioms should live in the same module as their class of declaration. The full axiomatisation of a class is now only provided if the module oeo-shared-axioms is used. See also the wiki article on the [modules of the OEO](https://github.com/OpenEnergyPlatform/ontology/wiki/Modules-of-the-OEO).

> Further, the import process for UO and OMO where updated. And any scripts and tools not essential for the building routine moved to a [seperate repository](https://github.com/OpenEnergyPlatform/oeo-tools).

Reviewer: ```suggestion
Further, the import process for UO and OMO where updated. All scripts and tools not essential for the building routine moved to a [seperate repository](https://github.com/OpenEnergyPlatform/oeo-tools). And finally, english language labels were added to (english) definitions and labels, to allow other language extentions in the future.
```
